### PR TITLE
Removed unnecessary updates of tab bar stylesheets

### PR DESCRIPTION
**Pull request by jacek-izykowski**

### CODE REVIEW

File: src/ui/dockwidget.cc [hunk 1]
```diff
@@ -1161,14 +1161,19 @@ void MainWindowWithDetachableDockWidgets::updateActiveDockWidget() {
...
           "color : palette(highlighted-text);"
           "border-bottom: 0px solid palette(shadow);"
-          "}"));
+          "}");
+      if(tab_bar->styleSheet() != stylesheet) {
+        tab_bar->setStyleSheet(stylesheet);
+      }
     } else {
-      tab_bar->setStyleSheet(tab_bar_stylesheet);
+      if(tab_bar->styleSheet() != tab_bar_stylesheet) {
```
Comment (mkow) [inline src/ui/dockwidget.cc]: Space after `if` please ;)

File: src/ui/dockwidget.cc [hunk 2]
```diff
@@ -1161,14 +1161,19 @@ void MainWindowWithDetachableDockWidgets::updateActiveDockWidget() {
...
         && active_dock_widget_) {
-      tab_bar->setStyleSheet(tab_bar_stylesheet +
+      QString stylesheet = tab_bar_stylesheet +
           QString("QTabBar::tab:selected {"
           "background : palette(highlight);"
           "color : palette(highlighted-text);"
           "border-bottom: 0px solid palette(shadow);"
-          "}"));
+          "}");
+      if(tab_bar->styleSheet() != stylesheet) {
```
Comment (mkow) [inline src/ui/dockwidget.cc]: space after `if`

Comment (jacek-izykowski) [inline src/ui/dockwidget.cc]: Changed.

File: src/ui/dockwidget.cc (hunk 1 above)
Comment (jacek-izykowski) [inline src/ui/dockwidget.cc]: Changed.

**Pull request merged**